### DISCUSSION TITLE
0.0.8pre

### DISCUSSION THREAD

**Discussion by OStrama** [OStrama/weishaupt_modbus#13] [Show and tell]
# Starting with version 0.0.8 we will start to consolidate both versions. 
# In version 0.0.8 this will have the following impact:

## For users of MadOne's original weishaupt_modbus integration:
 * When doing nothing than simply installing the integration, the long term statistics will be split into new entities,
   since the sensor domain is different.
 * To avoid this, edit the file "const.py" in the integration's directory as follows:
   replace the string
     PREFIX = "weishaupt_wbb"
   by
     PREFIX = "weishaupt_modbus"
   please do not change the intents or any other parts of the file to avoid issues

## For users of OStrama's weishaupt_wbb integration:
 *  Uninstall existing "weishaupt_wbb" installation, answer "integration and all entities of it will be deleted" with"yes"
 *  Restart home assistant
 *  Install new weishaupt_wbb integration
 *  You will get a new integration with the same name
 *  the sensor entities will be the same than before

<hr /><em>This discussion was created from the release <a href='https://github.com/OStrama/weishaupt_modbus/releases/tag/0.0.8pre'>0.0.8pre</a>.</em>

**Comment by thetschulian** [comment]
Hey ostrama,
would you recommend to installed the newest Pre release as soon as it appears or should we wait for the next "official" release?

Thanks for you time and effort.

**Comment by OStrama** [comment]
The pre-releasis are slightly tested in devcontainer and my HOme assistant. I use them in my real home assistant. I basically make them for easy installation. 
they do not contain everything that we plan for the release, but you can install them, too. 
Some feedback about issues would be nice;-)

**Reply by thetschulian** [reply]
sure, I will keep u update to date. since my WBB20 is running since less than 4 weeks, I am pretty sure I have some time to deal with the WBB20 in future so the HA integration is mandatory .

**Comment by OStrama** [comment]
1.0.4 is out, so I close this thread..
